Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.controller.ts

```diff
@@ -0,0 +1,28 @@
+﻿module Kitos.ItSystem.Usage.Tabs {
+    "use strict";
+
+    export class ItSystemUsageTabDataProcessingController {
+        static $inject: Array<string> = [
+            "itSystemUsage"
+        ];
+
+        constructor(
+            private readonly itSystemUsage) {
+
+            this.dataProcessingRegistrations = this.itSystemUsage.associatedDataProcessingRegistrations;
+        }
+
+        dataProcessingRegistrations: Models.OData.DataProcessing.IDataProcessingRegistration[];
```

Comment:
That's not the model used, since the itSystemUsage is not from the odata api.

I would suggest change it to use NamedEntityDTO, which defines the two fields you are using in the view (`id` and `name`)